server/bundler/utils.js

```diff
@@ -23,7 +31,36 @@ function pathToRegExp( path ) {
 	return new RegExp( '^' + path + '(/.*)?$' );
 }
 
+function hashFile( path ) {
+	const md5 = crypto.createHash( 'md5' );
+	let data, hash;
```

Comment:
Super minor but there is no need to declare `data` as it's only used inside the `try` clause.

Reply:
This function was not authored here (just moved)